pynest/nest/lib/hl_api_helper.py

```diff
@@ -472,23 +472,33 @@ def show_help_with_pager(hlpobj, pager):
 
     # reading ~/.nestrc lookink for pager to use.
     if pager is None:
-        # open ~/.nestrc
-        rc = open(os.environ['HOME'] + '/.nestrc', 'r')
-        for line in rc:
-            rctst = re.match(r'^\s?%', line)
-            if rctst is None:
-                pypagers = re.findall(
-                    r'\s?/page\s?<<\s?/command\s?\((.*)\).*', line)
-                if pypagers:
-                    pager = pypagers[0]
-                    break
-                else:
-                    pager = 'less'
-        rc.close()
-
+        # check if .netsrc exist
+        rc_file = os.environ['HOME'] + '/.nestrc'
+        if os.path.isfile(rc_file):
+            # open ~/.nestrc
+            rc = open(os.environ['HOME'] + '/.nestrc', 'r')
+            for line in rc:
+                rctst = re.match(r'^\s?%', line)
```

Comment:
Could you add a comment explaining/illustrating briefly what this RE looks for; that will make the code more maintainable. Applies also to other REs.

I think it would also be helpful if you could explain what this entire loop is trying to do---I cannot quite get it just from the code.